packages/tinacms/src/hooks/use-graphql-forms.tsx

```diff
@@ -136,6 +136,7 @@ export function useGraphqlForms<T extends object>({
   }, [pendingReset])
 
   React.useEffect(() => {
```

Comment:
I can't make an inline suggestion this far away from changes apparently, but we should add a `setIsLoading(false)` into the `.catch` block as well

Reply:
👍 Done